response_selectors/convers_evaluation_based_selector/README.md

```diff
@@ -36,6 +38,20 @@ LANGUAGE: language to consider
 FALLBACK_FILE: a file name with fallbacks from `dream/common/fallbacks/`
 ```
 
-## Dependencies
+## Input/Output
+**Input:** a list of hypotheses with their scores and metadata
+**Output:** a list of selected hypotheses with their scores and metadata
```

Comment:
not a list but the final hypothesis actually

Reply:
Fixed to a correct updated version.